packages/neos-ui-editors/src/Library/AssetUpload.js

```diff
@@ -73,12 +73,22 @@ export default class AssetUpload extends PureComponent {
         });
     }
 
-    getUploadMetaData() {
+    getUploadMetaData = () => {
         return this.props.imagesOnly ? 'Image' : 'Asset';
     }
 
+    acceptedFileTypes = () => {
+        if (this.props.imagesOnly) {
+            return ['.jpg', '.jpeg', '.png'].reduce((fileTypeString, fileType) => {
```

Comment:
I'm not sure which image formats I should allow here. Do we have a definition somewhere?

We could/should make it configurable later on I think.

Reply:
Well, seeing SVG more and more. Absolutely needs to be configurable. What about jp2, webm, …?

Reply:
Good question.
I work on a configurable solution soon.